check_zfs/check_zfs.py

```diff
@@ -0,0 +1,459 @@
+#!/usr/bin/python
```

Comment:
Same as the other comment. If python3 can be used please change this to `#!/usr/bin/python3`